lib/readme.txt

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
I don't think we should add this file. There is already a readme in the lib folder which has more important and more relevant content.